tests/MongoDB.Analyzer.Tests.Common.ClassLibrary/MongoDB.Analyzer.Tests.Common.ClassLibrary.csproj

```diff
@@ -11,6 +11,10 @@
     <PackageReference Include="MongoDB.Driver" Version="2.11.0">
          <ExcludeAssets>runtime</ExcludeAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.2.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="Microsoft.CodeAnalysis.CSharp" />
```

Comment:
What is the reason for this?

Reply:
I recall that the MongoDB.Analyzer.Tests.Common.ClassLibrary project had a build error concerning Microsoft.CodeAnalysis.CSharp. This has been reverted in the most recent changes